### ISSUE TITLE
Computed field

### ISSUE THREAD

**Issue by Keko-94** [dcasia/column-toggler#13]
Hello

It doesn't work with computed fields.
```
Text::make(__('Enterprise'), function () {
    return $this->training->enterprise->name;
})
```
Thanks for this package !

**Comment by milewski** [comment]
It is because computed fields have no attribute! perhaps you can use `->displayUsing()` instead

**Comment by milewski** [comment]
I figured out a way to make it work, released here: [v0.1.1](https://github.com/dcasia/column-toggler/releases/tag/v0.1.1)
